test/unit/api/autorequire-pouchdb-replication-test.js

```diff
@@ -0,0 +1,26 @@
+var test = require('tap').test
+var proxyquire = require('proxyquire').noPreserveCache()
+var state = {}
+
+state.PouchDB = function () {
+  return {}
+}
+
+state.PouchDB.plugin = function () {}
+
+test('Store replicate function', function (group) {
+  group.test('Returns a promise rejection when the module is not found', function (t) {
+    var replicate = proxyquire('../../../api/store/replicate', {'pouchdb-replication': null})
+    t.plan(1)
+    replicate(state)
+      .then(function () {
+        t.fail('Replicate function should not return a fulfilled promise')
+        t.end()
+      })
+      .catch(function (error) {
```

Comment:
can you check for the correct error message to make sure the error is not caused by something else?